modules/standard/IO.chpl

```diff
@@ -609,10 +612,12 @@ proc stringStyleWithVariableLength() {
 }
 
 /*
-  This method returns the appropriate :record:`iostyle` ``str_style`` value
-  to indicate a string format where string data is preceded by a ``lengthBytes``
-  of length. Only lengths of 1, 2, 4, or 8 are supported; if this method
-  is called with any other length, it will throw an error.
+  Return the appropriate :record:`iostyle` ``str_style`` value
+  to indicate a string format where string data is preceded by a `lengthBytes`
```

Comment:
Why did you remove the double ` here?

Reply:
We tend to put the names of formals in single backticks.